About.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="About.css">
```

Comment:
- [ ] Nice working on your About us page, Kindly check the link to the style sheet, the page is not loading properly, the `stylesheet` is in the style folder, the link should be `style/About.css`